src/wfs/src/main/java/org/geoserver/wfs/InsertElementHandler.java

```diff
@@ -235,8 +237,12 @@ public void execute(
             // update the insert counter
             inserted += featureList.size();
         } catch (Exception e) {
-            String msg = "Error performing insert: " + e.getMessage();
-            throw new WFSTransactionException(msg, e, insert.getHandle());
+            final StringBuilder msgBuilder = new StringBuilder();
+            msgBuilder.append("Insert error: ");
+            msgBuilder.append(e.getMessage());
+            final String exceptionMessage = msgBuilder.toString();
+            throw exceptionFactory.newWFSTransactionException(
+                    exceptionMessage, e, insert.getHandle());
```

Comment:
This change is functionally 1:1 since the Java compiler will create a StringBuilder for the + operation, so it's the same bytecode (and no extra objects with the former / same performance). But I fully agree that these simple cases will be more compact with + and thus more readable with fewer lines - I will update according to your suggestion (for all cases).

Reply:
PR branch is now updateed according to the review comments.